partition/roles/sonic/handlers/main.yaml

```diff
@@ -1,8 +1,4 @@
 ---
-- name: reload systemd
```

Comment:
this name is nowhere else used ?

Reply:
Is unused in the role `sonic` after moving affected parts to `metal-core`. In the role `metal-core`, I renamed the handler to `Reload systemd daemon`.